sycl/tools/sycl-check.cpp

```diff
@@ -1,299 +0,0 @@
-//==----------- sycl-check.cpp ---------------------------------------------==//
```

Comment:
Is this tool never used in any guides?

Reply:
It was added by @romanovvlad and I don't see any other mentions.